File: Java_Assignment/Assignment_56/programAQ4.java
///////////////////////////////////////////////////////////
//
//  File name : Assignement56Question4.java
//  Description : Write a java program which accept string and one character from user
//                and remove that character from string.
//  Author : Kartik Ganesh Jare
//  Date : 13/11/25
//  
///////////////////////////////////////////////////////////

/*

Input : IndiaisDemoIndia

i

Output : IndasDemoInda

*/

import java.util.Scanner;

class MyString
{
    public String Removechar(String str,char ch) 
    {
        String sResult = "";

        for(int i = 0;i < str.length();i++)
        {
            if(str.charAt(i) != ch)
            {
                sResult = sResult + str.charAt(i);
            }
        }

        return sResult;
    }
}

public class programAQ4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter frist string : ");
        String s1 = sobj.nextLine();

        System.out.println("Enter character : ");
        char ch = sobj.next().charAt(0);

        MyString aobj = new MyString();

        String sRet = aobj.Removechar(s1,ch);

        System.out.println(sRet);

        sobj.close();
    }   
}

File: C_Assignment/Assignment_31/programAQ4.c
///////////////////////////////////////////////////////////
//
//  File name : Assignement31Question4.c
//  Description : Write a program which accept one number from user and toggle 7th and
//                10th bit of that number. Return modified number.
//  Author : Kartik Ganesh Jare
//  Date : 13/11/25
//  
///////////////////////////////////////////////////////////

/*

Input : 137
Output : 713

*/

#include<stdio.h>

typedef unsigned int UINT;

UINT ToggleBit(UINT iNo)
{
    UINT iMask = 0;
    UINT iResult = 0;

    iMask = (1 << 6) | (1 << 9);
    iNo = iNo ^ iMask;

    return iNo;
}

int main()
{
    UINT iValue = 0;
    UINT iRet = 0;

    printf("Enter number: \n");
    scanf("%u",&iValue);

    iRet = ToggleBit(iValue);

    printf("changed number is : %u\n",iRet);


    return 0;
}